template/config/storybook/components/playground.vue

```diff
@@ -0,0 +1,153 @@
+<script lang="ts">
+import { PropOptions, VueConstructor } from 'vue';
+import { Component, Prop, Vue } from 'vue-property-decorator';
+import { StoryProp } from '.storybook/modules';
+// import { StoryProp } from '@/story';
+
+
+@Component
+export default class Playground extends Vue {
+  // --------------------------------------------------------------------------
+  // Fields
+  // --------------------------------------------------------------------------
+
+  @Prop({ type: Object })
+  public items!: { [key: string]: PropOptions<object> };
+
+  @Prop({ type: Function })
+  public source!: VueConstructor;
+
+  private readonly DEFAULT = "__undefined__";
+  private propNames: string[] = [];
+  private livePreview?: Vue = undefined;
+
+  // --------------------------------------------------------------------------
+  // Methods
+  // --------------------------------------------------------------------------
+
+  private mounted() {
+    const render = this.$refs.render as Element;
+    const livePreview = new this.source();
+
+    livePreview.$mount();
+    render.appendChild(livePreview.$el);
+
+    this.propNames = Object.keys(this.items);
+    this.livePreview = livePreview;
+  }
+
+  private getDefault(propName: string) {
+    return this.items[propName]?.default;
+  }
+
+  private getType(propName: string) {
+    // tslint:disable-next-line:ban-types
```

Comment:
@taslangraham all the tslint stuff i would recommend either adjusting the tslint.json file instead of disabling the rule inline. Also look into fixing these issues when the code linter raises them.

Reply:
@taslangraham  i didn't see any changes to the tslint.json or tsconfig.json file that would include the rules that are currently being disabled in the code using the // tslint comment. Any changes to push here?